docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -169,4 +469,112 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use JSON Schema file ``schema.jsonschema`` with content
+
+.. code:: json
+
+  {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "values": {
+        "type": "object",
+        "properties": {
+          "greeting": {
+            "type": "string"
+          },
+          "recipient": {
+            "type": "string"
+          }
+        },
+        "required": ["greeting", "recipient"],
+        "additionalProperties": false
+      }
+    },
+    "required": ["values"],
+    "additionalProperties": false
+  }
+
+and YAML file ``values.yaml`` with content
+
+.. code:: sh
+
+  values:
+    greeting: Hello
+    recipient: World
+
+* Validate a YAML config against a given JSON schema:
+
+  .. code:: sh
+
+    $ uw config validate --schema-file schema.jsonschema --input-file values.yaml
+    [2024-01-03T17:23:07]     INFO 0 UW schema-validation errors found
+
+  Shell redirection via ``|``, ``>``, et al may also be used to stream output to a file, another process, etc.
+
+
+* Read the config from ``stdin`` and print validation results to ``stdout``:
+
+  .. code:: sh
+
+    $ cat values.yaml | uw config validate --schema-file schema.jsonschema
+    [2024-01-03T17:26:29]     INFO 0 UW schema-validation errors found
+
+
+* However, reading the schema from ``stdin`` is **not** supported:
+
+  .. code:: sh
+
+    $ cat schema.jsonschema | uw config validate --input-file values.yaml
+    uw config validate: error: the following arguments are required: --schema-file
+
+* If a config fails validation, differences from the schema will be displayed. For example, with ``recipient: World`` removed from ``values.yaml``:
+
+  .. code:: sh
+
+    $ uw config validate --schema-file schema.jsonschema --input-file values.yaml
+    [2024-01-03T17:31:19]    ERROR 1 schema-validation error found
```

Comment:
Does this also change to `UW schema-validation...`? How about the rest of the ones below?

Reply:
Oops, missed that one - I'll fix it and recheck the full set